includes/organizer/organizer-cpt.php

```diff
@@ -46,12 +46,13 @@ function create_organizer_cpt()
         'menu_position' => 5,
         'show_in_admin_bar' => true,
         'show_in_nav_menus' => true,
+        'show_in_rest' => true,
         'can_export' => true,
         'has_archive' => true,
         'exclude_from_search' => false,
         'publicly_queryable' => true,
         'capability_type' => 'post',
         'menu_icon' => 'dashicons-groups',
     );
-    register_post_type('organizer', $args);
+    register_post_type('organisationer', $args);
```

Comment:
Based on my previous comments, the ideal for this should be:
`register_post_type('organisation', $args);` assuming that we keep it in UK-English